statsmodels/tsa/adfvalues.py

```diff
@@ -383,5 +383,387 @@ def mackinnoncrit(N=1, regression ="c", nobs=inf):
     else:
         return polyval(eval("tau_"+reg+"_2010["+str(N-1)+",:,::-1].T"),1./nobs)
 
+
+tau_max = {'nc': [inf, 1.51, 0.86, 0.88, 1.05, 1.24],
```

Comment:
These look identical to `tau_max_nc`, `tau_max_c`,... above.  Is that right?  The dictionary version here is much nicer, and should ideally be used to get rid of the `eval` kludge in `mackinnonp` and `mackinnoncrit`.